recipes/qimage2ndarray/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: qimage2ndarray
+  version: "1.5.1"
+
+source:
+  fn: qimage2ndarray-1.5.1.tar.gz
+  url: https://pypi.python.org/packages/source/q/qimage2ndarray/qimage2ndarray-1.5.1.tar.gz
+  md5: 73743819ccf6043f237c6b7f63732674
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - sip
+    - pyqt
+
+  run:
+    - python
+    - numpy
+    - sip
+    - pyqt
+
+test:
+  imports:
+    - qimage2ndarray
+
+about:
+  home: https://github.com/hmeine/qimage2ndarray
+  license: BSD License
+  summary: 'Conversion between QImages and numpy.ndarrays.'
+
```

Comment:
I wouldn't normally be so picky, but since you will be making tweaks to the meta anyway, any change you'd mind removing the extra newline? #imtoopicky :wink: